package.json

```diff
@@ -1,27 +1,44 @@
-{
-  "name": "browser-game",
-  "version": "0.0.0",
-  "private": true,
-  "scripts": {
-    "devStart": "nodemon ./src/server.ts",
-    "start": "node ./build/server.js",
-    "build": "tsc"
-  },
-  "dependencies": {
-    "@types/mongoose": "^5.7.37",
-    "cookie-parser": "~1.4.4",
-    "debug": "~2.6.9",
-    "ejs": "~2.6.1",
-    "express": "~4.16.1",
-    "http-errors": "~1.6.3",
-    "mongoose": "^5.10.13",
-    "morgan": "~1.9.1"
-  },
-  "devDependencies": {
-    "@types/express": "^4.17.8",
-    "@types/morgan": "^1.9.2",
-    "nodemon": "^2.0.6",
-    "ts-node": "^9.0.0",
-    "typescript": "^4.0.5"
-  }
-}
+{
+  "name": "browser-game",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "devStart": "nodemon ./src/server.ts",
+    "start": "node ./build/server.js",
+    "build": "tsc",
+    "build-css": "postcss src/tailwind.css -o public/stylesheets/tailwind.css"
```

Comment:
you can change it to run both commands:
`"build": "tsc && postcss src/tailwind.css -o public/stylesheets/tailwind.css",`